cmd/minishift/cmd/start.go

```diff
@@ -117,6 +118,9 @@ var (
 		"host-pv-dir":      "/var/lib/minishift/openshift.local.pv",
 	}
 
+	// Set the base dir for v3.10.0
+	baseDirectory = "/var/lib/minishift/base"
+
```

Comment:
@praveenkumar  The code does not look clean here as we have both `dataDirectory` and `baseDirectory`. AFAIU we do not need  `dataDirectory`  in 3.10 and later. So is something we need to refactor after 3.12 or something?
```
// Set default value for host data and config dir
	dataDirectory = map[string]string{
		"host-config-dir":  "/var/lib/minishift/openshift.local.config",
		"host-data-dir":    "/var/lib/minishift/hostdata",
		"host-volumes-dir": "/var/lib/minishift/openshift.local.volumes",
		"host-pv-dir":      "/var/lib/minishift/openshift.local.pv",
	}
```

Reply:
@LalatenduMohanty yes this is something we will completely remove after 3.11 release then will have 3.10 and 3.11 supported only.